tools/prowgen/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM golang:1.17 as build-env
+
+WORKDIR /go/src/istio
+
+COPY . /go/src/istio
+
+RUN go install /go/src/istio/tools/prowgen/cmd/prowgen
+
+RUN go install github.com/mikefarah/yq/v4@v4.16.2
+
+FROM gcr.io/distroless/base:22bd467b41e5e656e31db347265fae118db166d9
```

Comment:
nit: use `static` instead of `base`

Reply:
Done.
That was copied from another Dockerfile, so I made the change together.